test/helpers/ssh_command.go

```diff
@@ -289,8 +265,10 @@ func (client *SSHClient) RunCommandContext(ctx context.Context, cmd *SSHCommand)
 	}
 	select {
 	case <-ctx.Done():
-		// Wait until the ssh session is stopped
-		wg.Wait()
+		log.Warning("sending SIGHUP to session due to canceled context")
+		if err = session.Signal(ssh.SIGHUP); err != nil {
```

Comment:
why not do the `session.Close` as well?

Reply:
We do, it's deferred above (lines 256-258).